planb/PBPackageInstaller.m

```diff
@@ -155,6 +161,19 @@ - (NSString *)downloadPackage {
   return path;
 }
 
+- (void)validatePackage:(NSString *)path {
+  NSString *checksum = [self SHA256ForFileAtPath:path];
```

Comment:
If self.checksum is not set, do you need to calculate this?

Reply:
Yes, just for logging though.  The previous version logged the SHA1 checksums of each package that it downloaded.  This changes it to log the SHA256 checksums instead since we were already computing them.

Reply:
ack